nohup: ignoring input
Using TensorFlow backend.
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:107: The name tf.reset_default_graph is deprecated. Please use tf.compat.v1.reset_default_graph instead.

WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:111: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:66: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

2023-05-26 17:05:34.470828: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
2023-05-26 17:05:34.503633: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2400000000 Hz
2023-05-26 17:05:34.531088: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5646bac2ea70 executing computations on platform Host. Devices:
2023-05-26 17:05:34.531131: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2023-05-26 17:05:34.568302: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:541: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:4432: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:3733: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:81: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 8, activation="relu", padding="same")`
  omics_conv1_ = Conv1D(128, 8, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:83: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 1, activation="relu", padding="same")`
  omics_conv2_ = Conv1D(64, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:85: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 3, activation="relu", padding="same")`
  omics_conv3_ = Conv1D(64, 3, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:87: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 1, activation="relu", padding="same")`
  omics_conv4_ = Conv1D(128, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:112: UserWarning: Update your `Model` call to the Keras 2 API: `Model(inputs=[<tf.Tenso..., outputs=[<tf.Tenso...)`
  model = Model(input=[input_seq, inputs_omics], output=[pred_output])
WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/keras/optimizers.py:793: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/lyli/miniconda3/envs/SeqPose/lib/python3.7/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
ACC: 0.721820809248555
Sens: 0.485177151120752
Spec: 0.9581227436823104
MCC: 0.5032238637752208
ROC_AUC: 0.7994560039259601
PRC_ROC: 0.8370723149663688
WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:81: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 8, activation="relu", padding="same")`
  omics_conv1_ = Conv1D(128, 8, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:83: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 1, activation="relu", padding="same")`
  omics_conv2_ = Conv1D(64, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:85: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 3, activation="relu", padding="same")`
  omics_conv3_ = Conv1D(64, 3, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:87: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 1, activation="relu", padding="same")`
  omics_conv4_ = Conv1D(128, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:112: UserWarning: Update your `Model` call to the Keras 2 API: `Model(inputs=[<tf.Tenso..., outputs=[<tf.Tenso...)`
  model = Model(input=[input_seq, inputs_omics], output=[pred_output])
ACC: 0.7250722543352601
Sens: 0.48988439306358383
Spec: 0.9602601156069365
MCC: 0.5100982722733302
ROC_AUC: 0.8109651174446189
PRC_ROC: 0.8446618331837853
WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:81: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 8, activation="relu", padding="same")`
  omics_conv1_ = Conv1D(128, 8, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:83: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 1, activation="relu", padding="same")`
  omics_conv2_ = Conv1D(64, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:85: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 3, activation="relu", padding="same")`
  omics_conv3_ = Conv1D(64, 3, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:87: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 1, activation="relu", padding="same")`
  omics_conv4_ = Conv1D(128, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:112: UserWarning: Update your `Model` call to the Keras 2 API: `Model(inputs=[<tf.Tenso..., outputs=[<tf.Tenso...)`
  model = Model(input=[input_seq, inputs_omics], output=[pred_output])
ACC: 0.7286849710982659
Sens: 0.5180635838150289
Spec: 0.9393063583815029
MCC: 0.5042957080797801
ROC_AUC: 0.8059524729359485
PRC_ROC: 0.8434042539907664
WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:81: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 8, activation="relu", padding="same")`
  omics_conv1_ = Conv1D(128, 8, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:83: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 1, activation="relu", padding="same")`
  omics_conv2_ = Conv1D(64, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:85: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 3, activation="relu", padding="same")`
  omics_conv3_ = Conv1D(64, 3, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:87: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 1, activation="relu", padding="same")`
  omics_conv4_ = Conv1D(128, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:112: UserWarning: Update your `Model` call to the Keras 2 API: `Model(inputs=[<tf.Tenso..., outputs=[<tf.Tenso...)`
  model = Model(input=[input_seq, inputs_omics], output=[pred_output])
ACC: 0.7232658959537572
Sens: 0.5025270758122744
Spec: 0.9443239334779465
MCC: 0.49800938024976316
ROC_AUC: 0.8075835767480835
PRC_ROC: 0.8375580993352569
WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:81: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 8, activation="relu", padding="same")`
  omics_conv1_ = Conv1D(128, 8, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:83: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 1, activation="relu", padding="same")`
  omics_conv2_ = Conv1D(64, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:85: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(64, 3, activation="relu", padding="same")`
  omics_conv3_ = Conv1D(64, 3, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:87: UserWarning: Update your `Conv1D` call to the Keras 2 API: `Conv1D(128, 1, activation="relu", padding="same")`
  omics_conv4_ = Conv1D(128, 1, activation='relu', border_mode='same')
/data/lyli/Silencer/K562/datasets/6924/5fold/model/checkpoints_4layer_100-/Model_CNN.py:112: UserWarning: Update your `Model` call to the Keras 2 API: `Model(inputs=[<tf.Tenso..., outputs=[<tf.Tenso...)`
  model = Model(input=[input_seq, inputs_omics], output=[pred_output])
ACC: 0.729407514450867
Sens: 0.5826714801444043
Spec: 0.8763557483731019
MCC: 0.4801517118887184
ROC_AUC: 0.8147823885186549
PRC_ROC: 0.845025572858016
ACC_average: 0.725650289017341
Sens_average: 0.5156647367912087
Spec_average: 0.9356737799043596
MCC_average: 0.4991557872533625
ROC_average: 0.8077479119146531
PRC_average: 0.8415444148668387

